src/dataExplorer/components/MeasurementSelector.tsx

```diff
@@ -0,0 +1,56 @@
+import React, {FC, useContext, useMemo, useState} from 'react'
+import {isEmpty} from 'lodash'
+
+// Components
+import {ComponentStatus} from '@influxdata/clockface'
+import SelectorTitle from 'src/dataExplorer/components/SelectorTitle'
+import SearchableDropdown from 'src/shared/components/SearchableDropdown'
+
+// Context
+import {NewDataExplorerContext} from 'src/dataExplorer/context/newDataExplorer'
+
+const MeasurementSelector: FC = () => {
+  const {
+    measurements,
+    selectedBucket,
+    selectedMeasurement,
+    selectMeasurement,
+  } = useContext(NewDataExplorerContext)
+  const [searchTerm, setSearchTerm] = useState('')
+
+  const handleSelect = (option: string): void => {
+    selectMeasurement(option)
+  }
+
+  const handleChangeSearchTerm = (value: string) => {
+    setSearchTerm(value)
+  }
+
+  return useMemo(() => {
+    if (isEmpty(selectedBucket)) {
+      return null
+    }
```

Comment:
preference for not using lodash methods here. If there's a way we can derive this natively that would be 🔥